test/e2e/framework/device_plugin_utils.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1alpha"
+)
+
+const (
+	// Dummy device resource name to be advertised to Kubelet
+	StubDeviceName = "dummy.device.com/device"
+
+	// stub device plugin image
+	StubDevicePluginImage = "gcr.io/kubernetes-e2e-test-images/stub-device-plugin-amd64:1.0"
```

Comment:
Instead of adding here, please add an entry [here](https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L84) and use [GetE2EImage](https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L86) to act like a multiarch image

Reply:
Thanks, Done.